src/linkInfo.ts

```diff
@@ -13,11 +13,12 @@ export class LinkInfo {
     public toLink(): string {
         var startIndex = this.position.start.line + 1;
         var endIndex = this.position.end.line + 1;
-        var prefix = `https://${this.siteName}/${this.userName}/${this.repoName}/blob/${this.branchName}/${this.filePath}`;
+        var siteUrlSource = (this.siteName.split("@")[1] === "bitbucket.org") ? 'src' : 'blob';
```

Comment:
I debugged the extension within vscode and siteName was returning something like myUserName@siteDomain, (where myUserName is your git username and siteDomain is the remote domain), and to compare if the remote URL was from bitbucket or from github i splitted siteName to compare only the domain...